src/genegraph/source/graphql/gene_feature.clj

```diff
@@ -15,10 +15,10 @@
   )
 
 (defn previous-symbols [context args value]
-  )
+  (str/join ", " (q/ld-> value [:skos/hidden-label])))
 
 (defn alias-symbols [context args value]
-  (str/join ", " (q/ld-> value [:skos/hidden-label])))
```

Comment:
Same as above, though I think I may have made a mistake in putting too much meaning into the SKOS terms by this point. Probably the right answer is to assign the aliases to a special attribute just for genes...

Reply:
Help me understand the reasoning here... are you saying that skos/hidden-label casts too wide a net, and you feel that we should use one specific to aliases?